developer-workflow/c-api.rst

```diff
@@ -4,18 +4,20 @@
 Changing Python's C API
 =======================
 
-The C API is divided into three sections:
+The C API is divided into these tiers:
 
 1. The internal, private API, available with ``Py_BUILD_CORE`` defined.
    Ideally declared in ``Include/internal/``. Any API named with a leading
    underscore is also considered private.
-2. The public C API, available when ``Python.h`` is included normally.
+2. The “general” public C API, available when ``Python.h`` is included normally.
    Ideally declared in ``Include/cpython/``.
-3. The Limited API, available with ``Py_LIMITED_API`` defined.
+3. The Unstable C API, identified by the ``PyUnstable_`` name prefix.
+   Ideally declared in :cpy-file:`Include/cpython/` along with the general public API.
+4. The Limited C API, available with :c:macro:`Py_LIMITED_API` defined.
    Ideally declared directly under ``Include/``.
 
-Each section has higher stability & maintenance requirements, and you will
-need to think about more issues when you add or change definitions in it.
+Each tier has different stability & maintenance requirements to think about
```

Comment:
nitpick
```suggestion
Each tier has different stability & maintenance requirements to consider
```

Reply:
Slightly more conventional/idiomatic, though small and subjective enough that I didn't mention it. But if we're going to nitpick anyway, typically "&" is only used as a replacement for "and" in English in [very specific contexts](https://en.wikipedia.org/wiki/Ampersand#Usage and carries a somewhat distinct meaning